js/viewer/attributes.js

```diff
@@ -0,0 +1,55 @@
+/*globals */
+
+OC.Plugins.register('OC.Share.ShareItemModel', {
+	attach: function (model) {
+
+		if (_.isUndefined(model)) {
+			throw 'missing OC.Share.ShareItemModel';
+		}
+
+		// Register additional available share attributes
+		var mimetype = model.getFileInfo().getMimeType();
+		var folderMimeType = 'httpd/unix-directory';
+		if ((odfViewer.isSupportedMimeType(mimetype) || mimetype === folderMimeType) &&
+			OC.appConfig.richdocuments &&
+			OC.appConfig.richdocuments.secureViewAllowed &&
+			OC.appConfig.richdocuments.defaultShareAttributes) {
+			/** @type OC.Share.Types.RegisteredShareAttribute **/
+			var secureViewCanDownload = {
+				"scope": "permissions",
+				"key": "download",
+				"default": JSON.parse(OC.appConfig.richdocuments.defaultShareAttributes.secureViewCanDownload),
+				"label": t('richdocuments', 'can view without watermark'),
+				"description": t('richdocuments', 'With this permission being' +
```

Comment:
from what I see the "description" field is not used in the core PR, so please remove it.